rdmo/core/settings.py

```diff
@@ -295,6 +295,8 @@
 
 EXPORT_CONTENT_DISPOSITION = 'attachment'
 
+EXPORT_MIN_REQUIRED_VERSION = '2.1.0'
```

Comment:
this should only be changed when new RDMO versions would bring breaking changes, a backwards compatibility method should be added to support the previous versions but newly exported files can only be imported by the new RDMO version.